### PR TITLE
Device store dimensions from rootview

### LINKED ISSUE #278: Split view doesn't work with correct dimensions
See: https://slack-files.com/T024Z3H2Y-F93TJECJX-c73e2d3cd4 (it should be responsive - change to mobile view). I tried to investigate a little bit few weeks ago but it seems like it's broken in RN: https://github.com/facebook/react-native/issues/16152

### PR DESCRIPTION
**Pull request by tbergq**
closes #278

### CODE REVIEW

File: app/shared/src/Device.js [hunk 1]
```diff
@@ -2,39 +2,55 @@
...
   },
   isLandscape() {
-    const { height, width } = renewDimensions();
+    const { height, width } = this.renewDimensions();
     return width >= height;
   },
   isTablet() {
-    if (Platform.OS === 'ios' && Platform.isPad) {
-      return true;
-    }
```
Comment (mrtnzlml) [inline app/shared/src/Device.js]: Why did you remove this condition? Maybe we should refactor it a little bit more - tablet is still tablet even though it's in the multitasking mode and it's a mistake we are adjusting the design based this I think.

Comment (tbergq) [reply app/shared/src/Device.js]: Maybe, now isTable is used to decide if we should show map with hotel list, render map view icon in header etc. Maybe we should add another method to decide if we should show map with hotel list?

Comment (mrtnzlml) [reply app/shared/src/Device.js]: The logic of showing the icon is wrong. It doesn't depend on the tablet/phone but on the available width. Let me think about it for a while. I am not very happy with this implementation yet... 🙂
